File: app/src/main/java/com/example/android/test1/DataBase/AppDatabase.java
package com.example.android.test1.DataBase;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

import com.example.android.test1.DataBase.WeatherDAO;
import com.example.android.test1.POJO.WeatherDB;

@Database(entities = {WeatherDB.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
        public abstract WeatherDAO weatherDAO();

    }

File: app/src/main/java/com/example/android/test1/DataBase/AppDatabase.kt
package com.example.android.test1.DataBase

import androidx.room.Database
import com.example.android.test1.data.WeatherDB
import androidx.room.RoomDatabase

@Database(entities = [WeatherDB::class], version = 1)
abstract class AppDatabase : RoomDatabase() {

    public abstract fun weatherDAO(): WeatherDao
}